ang/civicase/ContactCaseTabList.js

```diff
@@ -0,0 +1,39 @@
+(function (angular, $, _) {
+  var module = angular.module('civicase');
+
+  module.directive('civicaseContactCaseTabList', function () {
```

Comment:
Please name this as `civicaseContactCaseTabCaseList `. and change file names and css naming as well.